src/app/migration/src/src/app/modules/core/services/user/user.service.ts

```diff
@@ -184,6 +185,29 @@ export class UserService {
     this._userid = this._userProfile.userId;
     this._rootOrgId = this._userProfile.rootOrgId;
     this._userData$.next({ err: null, userProfile: this._userProfile });
+    this.setRoleOrgMap(profileData);
```

Comment:
call it before this._userData$.next({ err: null, userProfile: this._userProfile });